crates/frontend/src/components/table/types.rs

```diff
@@ -10,11 +11,22 @@ pub struct TableSettings {
     pub redirect_prefix: Option<String>,
 }
 
+#[derive(Clone)]
+pub enum ColumnSortable {
+    Yes {
+        sort_fn: Callback<()>,
+        sort_by: SortBy,
+        current: bool,
```

Comment:
```suggestion
        sorted: bool,
```
I am assuming `current` is for whether the respective column is the one according to which the table data is sorted.

Reply:
How about currently_sorted?